pulp_smash/tests/pulp3/utils.py

```diff
@@ -290,3 +290,34 @@ def get_repo_versions(repo):
         [version['_href'] for version in versions],
         key=lambda url: int(urlsplit(url).path.split('/')[-2])
     )
+
+
+def get_artifact_paths(repo, version_href=None):
+    """Return the paths of artifacts present in a given repository version.
+
+    :param repo: A dict of information about the repository.
+    :returns: A list with the paths of units present in a given repository.
+    """
+    return set([
+        content_unit['artifact']  # file path and name
+        for content_unit in get_content(repo, version_href)['results']
+    ])
+
+
+def delete_repo_version(repo, version_href=None):
+    """Delete a given repository version.
+
+    :param repo: A dict of information about the repository.
+    :param version_href: The repository version that should be
+        deleted.
+    :returns: A dict of information about the last task created when deleting
+        a given repository version.
+    """
+    if version_href is None:
+        version_href = repo['_latest_version_href']
+    cfg = config.get_config()
+    client = api.Client(cfg, api.json_handler)
+    call_report = client.delete(version_href)
+    # As of this writing, Pulp 3 only returns one task. If Pulp 3 starts
+    # returning multiple tasks, this may need to be re-written.
+    return next(api.poll_spawned_tasks(cfg, call_report))
```

Comment:
See https://github.com/PulpQE/pulp-smash/issues/925.

Reply:
Alternately, let this function `return tuple(api.poll_spawned_tasks(cfg, call_report))`.